android-fast-dump/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# For more information about using CMake with Android Studio, read the
+# documentation: https://d.android.com/studio/projects/add-native-code.html
+
+set(TARGET fast-dump)
+project(${TARGET})
+
+# Sets the minimum version of CMake required to build the native library.
+cmake_minimum_required(VERSION 3.4.1)
+
+# Creates and names a library, sets it as either STATIC
+# or SHARED, and provides the relative paths to its source code.
+# You can define multiple libraries, and CMake builds them for you.
+# Gradle automatically packages shared libraries with your APK.
+add_library( # Sets the name of the library.
```

Comment:
Generally we prefer to have native projects compiled with the following flags:

    add_compile_options(-Wall -Wextra -Werror)

Reply:
done